commondata/CMS_t_sch_8TeV.yaml

```diff
@@ -1,9 +1,16 @@
 dataset_name: CMS_t_sch_8TeV
+doi: 10.1007/JHEP09(2016)027
+location: Sec. 9, combined
+arxiv: 1603.02555
+hepdata: none
+units: pb
+description: single top quark production cross-section in the s-channel
+luminosity: 19.7
 num_data: 1
 num_sys: 1
 data_central: 13.4
-statistical_error: 1.34
+statistical_error: 0
```

Comment:
was this a bug in the end?

Reply:
Yes it was! If you check Sec. 9 (combined result) in https://arxiv.org/pdf/1603.02555.pdf, you can see that it is correct this way.